runtime/src/anchor/mod.rs

```diff
@@ -118,7 +118,7 @@ decl_module! {
         /// # <weight>
         /// minimal logic, also needs to be consume less block capacity + cheaper to make the pre-commits viable.
         /// # </weight>
-        #[weight = 500_000]
+        #[weight = 193_000_000]
```

Comment:
Instead of constants, what do you think about using benchmarking as Substrate does to determine the time complexity, and using `T::DbWeight::get().reads_writes(1, 1)` for storage access? See https://centrifuge.hackmd.io/Mb8IAGhHQ32DEl1GIoBkCQ#Action-items for details.

Reply:
`T::DbWeight::get().reads_writes(1, 1)` implies we are doing only one read and one write for which is incorrect. Hence, I would like to keep our weights which were constant before as constants now to imply we are not considering the reads and writes but rather a chosen fixed weight @philipstanislaus

Reply:
@vedhavyas keeping the weights constant is a bad idea, since we most likely missprice our extrinsics compared to substrate. The issue with that is an attack vector – if you find a severely underpriced extrinsic, you could just send a load of them and the block can't be executed within the time limit (Not sure what happens then though?). 

Could you create a ticket to run benchmarks and use the actual reads/writes? Maybe link to that issue: https://github.com/paritytech/substrate/issues/6168